src/screens/LinkContextScreen/elements/Content.js

```diff
@@ -37,13 +37,13 @@ const styles = StyleSheet.create({
     paddingHorizontal: 20,
     marginTop: 12,
     marginBottom: 12,
-    color: COLORS.white2
+    color: COLORS.white
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect readability on different screens and light conditions. If `COLORS.white2` was a lighter shade, it might have been used for better contrast against certain backgrounds.

```diff
-    color: COLORS.white2
+    color: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->